_includes/html/seo.html

```diff
@@ -0,0 +1,15 @@
+<meta content="yes" name="apple-mobile-web-app-capable">
+<meta content="default" name="apple-mobile-web-app-status-bar-style">
+<meta name="description" content="{{ site.description }}">
```

Comment:
Won't this tag have empty content, as the `site.description` variable will have no value, as it was removed from [`_config.yml`](https://github.com/2factorauth/twofactorauth/pull/5695/files#diff-ecec67b0e1d7e17a83587c6d27b6baaaa133f42482b07bd3685c77f34b62d883)?

Same for the `site.title`, `site.url`, `site.author` variables which are used in this file and have also been removed from the `_config.yml` file?

Reply:
Actually the Jekyll-GitHub-metadata populates those variables automatically when building the site.